### ISSUE TITLE
why are there empty payment lines?

### ISSUE THREAD

**Issue by smadamit** [inbaz/ers#484]
0.3.51

screenshot from admin/matching/manual

![screen](https://cloud.githubusercontent.com/assets/1533081/8135783/d210f488-1139-11e5-9ec3-65ada65c757e.png)

**Comment by smadamit** [comment]
I disabled the strange lines, not only the completely empty ones: See admin/matching/disabled

See also id 2648 for example...

**Comment by inbaz** [comment]
I do not have the csv you uploaded into the system. I think there are empty lines, maybe at the end. In this part of the program only some of the fields are displayed maybe other fields are filled with any information. Can you have a look at the csv?

**Comment by smadamit** [comment]
see my email...

**Comment by smadamit** [comment]
OK there are this lines at the end of the csv. Can you filter such lines in the upload process? Maybe if one of the needed columns is empty.

```
,,,,,,,,
,,,,,,,,
,,,,,,,,
BIJ,315592.38,,,,,,,
AF,150352.96,,,,,,,
Total,,165239.42,,,,,,
,,,,,,,,
,,,,,,,,
BIJ CC,61169.38,,,,,,,
Bank Total,,104070.04,,,,,,
```

**Comment by smadamit** [comment]
fixed with #481
